package.json

```diff
@@ -35,15 +35,15 @@
   },
   "dependencies": {
     "@emotion/css": "^11.1.3",
-    "@grafana/data": "10.4.3",
+    "@grafana/data": "11.2.0",
     "@grafana/experimental": "^1.7.4",
     "@grafana/faro-web-sdk": "^1.7.0",
     "@grafana/faro-web-tracing": "^1.7.0",
-    "@grafana/flamegraph": "11.1.0-179306",
-    "@grafana/runtime": "10.4.3",
+    "@grafana/flamegraph": "11.2.0",
```

Comment:
I assume that the upgrade of the flame graph package has something to do with the new height of the E2E screenshots

Reply:
A lot of screenshots show an alert banner saying "Unauthorized". Could it be a regression?

Reply:
I think it's coming from 
http://localhost:3000/api/user/preferences enpoint

Reply:
It seems to be happening because of https://github.com/grafana/grafana/pull/89908. When the menu is opened it loads preferences (which was not happening in previous versions).

Reply:
Fixed in https://github.com/grafana/explore-profiles/pull/173/commits/eae8f8fd938041261b1d2df38d9d0440cde5601c